substrate/client/transaction-pool/tests/zombienet/mod.rs

```diff
@@ -152,7 +152,7 @@ impl NetworkSpawner {
 	}
 
 	/// Waits for blocks production/import to kick-off on given node.
```

Comment:
```suggestion
	/// Waits for blocks production/import to kick-off on given node.
	/// If `finalized` is true, the function will block till first finalized block is notified.
	/// Otherwise it will wait for the first best block.
```